config/config.go

```diff
@@ -28,6 +28,9 @@ type Config struct {
 	// For example, "gopkg.in/(.*)" => "github.com/\1" would translate
 	// gopkg into github (incorrectly, but the example would work).
 	Translate map[string]string `hcl:"translate,optional"`
+
+	// Allows to set url for a license (applies only when SBOM format is used as the output)
+	SBOMLicenseURLs map[string]string `hcl:"sbomLicenseURLs,optional"`
```

Comment:
I'm wondering if this couldn't be a generic thing for all outputs.
i.e. `licenseURLs`
Seems to be too specific to a particular output compared to other properties in this project.

Reply:
Also, I think the license URL could be extracted in a generic way.

in

`lciense/github/repo_api.go` you have a 
```
	rl, _, err := f.Client.Repositories.License(ctx, matches[1], matches[2])
```
which fetches the license information.
I think the `HTMLURL` field form `RepositoryLicense` (from `github.com/google/go-github`) should return the link to the license file found in the repo.

Extend the 

```
type License struct {
	Name string // Name is a human-friendly name like "MIT License"
	SPDX string // SPDX ID of the license, blank if unknown or unavailable
}
```
from `license/license.go`
with an additional URL attribute and fill it with data from `HTMLURL`.

This way we should have much more complete sbom without having to define all URLs in the config file.

Reply:
If we want to have `licenseURLs` flat in the config, then some changes should be made in xlsx as well to keep it consistent.

Below is the rule when licence URL should be added.

If the license can be resolved it will contain only SDPX ID.
Otherwise it will contain a name + license url.

For sure I don't want to define all URLs, only specific ones.

The changes you are proposing would work for zipstream dependency but for our deps on bitbucket, we would still need a way to set it.

Reply:
Here's the sbom xsd, if you want to a look at the license object: https://github.com/CycloneDX/specification/blob/master/schema/bom-1.1.xsd#L220